<launch>

    <node pkg="minimum_curvature" type="run.py" name="minimum_curvature" output="screen" respawn="true">

        <param name="frame" value="ugr/car_base_link" />
        <param name="rate" value="5" />

        <param name="carwidth" value="0.50" />
        <param name="width_margin_left" value="0.20" />
        <param name="width_margin_right" value="0.20" />
        <param name="carlength" value="0.72" />
        <param name="v_max" value="10.0" />
        <param name="drag_coeff" value="1.00" />
        <param name="car_mass" value="50.00" />
        <param name="plot" value="True" />
        <param name="imported_bounds" value="False" />
        <param name="curvlim" value="0.5" />
        <param name="stepsize_prep_trajectory" value="0.50" />
        <param name="stepsize_prep_boundaries" value="0.25" />
        <param name="stepsize_opt" value="0.25" />
        <param name="stepsize_post" value="0.25" />
        <param name="vel" value="True" />
        <param name="min_distance" value="1.5" />
        <param name="smoothing_factor_prep_trajectory" value="2.0" />
        <param name="smoothing_factor_prep_boundaries" value="0.1" />

        <remap from="/input/boundaries" to="/boundary_estimation/boundaries" />

        <remap from="/input/path" to="/pathplanning/poses" />
        <remap from="/output/path" to="/pathplanning/path" />
        <remap from="/output/path_extra" to="/minimum_curvature/path_extra" />
        <remap from="/output/path_iqp" to="/minimum_curvature/path_iqp" />
        <remap from="/output/path_ref" to="/minimum_curvature/path_ref" />
        
    </node>

</launch>